bigml/logistic.py

```diff
@@ -87,6 +87,22 @@ def get_unique_terms(terms, term_forms, tag_cloud):
     return terms_set.items()
 
 
+def balance_input(input_data, fields):
+    """Balancing the values in the input_data using the corresponding
+    field scales
+
+    """
+
+    for field in input_data:
+        if fields[field]['optype'] == 'numeric':
+            mean = fields[field]['summary']['mean']
+            stddev = fields[field]['summary'][ \
+                'standard_deviation']
```

Comment:
it's a corner case, but if you want to be paranoid here, it's not impossible that `mean` and `stddev` are `None`: that's what BigML returns for numeric fields that have all its values missing (until recently, that was even rarer than usual because all-missings fields were usually classified as text, while nowadays they can be numeric too).